src/js/client/controls.js

```diff
@@ -0,0 +1,150 @@
+// Present a standard set of user controls with login/logout/profile links
+// TODO: Convert this to use a common "toggler" grade once that has been sufficiently exercised and moved somewhere central.
+/* global fluid, jQuery */
+(function ($) {
+    "use strict";
+    var gpii = fluid.registerNamespace("gpii");
+    fluid.registerNamespace("gpii.express.user.frontend.controls");
+
+    gpii.express.user.frontend.controls.handleMenuKeys = function (that, event) {
```

Comment:
Up to date demo sample: https://github.com/fluid-project/infusion/blob/master/demos/keyboard-a11y/js/five-star.js#L121